bootstrap_salt/fab_tasks.py

```diff
@@ -32,6 +32,21 @@ def aws(x):
 
 
 @task
+def environment(x):
+    env.environment = str(x).lower()
+
+
+@task
+def application(x):
+    env.application = str(x).lower()
+
+
+@task
+def config(x):
+    env.config = str(x).lower()
```

Comment:
Ah what I suggested didn't work because we had done it wrong. https://github.com/ministryofjustice/bootstrap-cfn/pull/91 should make it so that the `--set` behaviour works again.